fpna/Utils/ExtentionMethods.ipynb

```diff
@@ -19,7 +19,28 @@
         {
             "cell_type": "markdown",
             "source": [
-                "# Basic Cashflow Computation"
+                "# Basic Cashflow Computation",
+                "\n## Compute Constant Cashflow",
+                "\nThis method will provide a constant cashflow of \"**Initial Value**\".",
+                "\n",
+                "\nThe cash flow starts after \"**shift**\" projection periods and is paid for \"**length**\" projection periods."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static double[] GetConstantValues(this double initialValue, int shift, int length){",
+                "\n",
+                "\n    var constant = new double[shift + length];",
```

Comment:
I find confusing having a length parameter which does not controls the length of the array.
Didn't we say that the array should indeed have Lenght = length?

Reply:
changed the meaning of "Length", including the description.

Reply:
changed the meaning of length